### PR TITLE
refactor: use humantime for user-facing options

### PR DESCRIPTION
**Pull request by tisonkun**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

See https://github.com/GreptimeTeam/greptimedb/pull/2709#discussion_r1387884581

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by killme2008 — APPROVED** [review]
LGTM. Thanks a lot.

File: config/metasrv.example.toml [hunk 1]
```diff
@@ -31,9 +31,9 @@ retry_delay = "500ms"
 # Failure detectors options.
 [failure_detector]
 threshold = 8.0
-min_std_deviation_millis = 100.0
-acceptable_heartbeat_pause_millis = 3000
-first_heartbeat_estimate_millis = 1000
+min_std_deviation = '100ms'
+acceptable_heartbeat_pause = '3000ms'
+first_heartbeat_estimate = '1000ms'
```
Comment (waynexia) [inline config/metasrv.example.toml:36]: sugg.(non-blocking): other quotes are double-quote. Though Toml can handle both, I'd prefer to unify them in one config file.
```suggestion
min_std_deviation = "100ms"
acceptable_heartbeat_pause = "3000ms"
first_heartbeat_estimate = "1000ms"
```

File: src/cmd/src/metasrv.rs [hunk 2]
```diff
@@ -219,9 +219,9 @@ mod tests {
             
             [failure_detector]
             threshold = 8.0
-            min_std_deviation_millis = 100.0
-            acceptable_heartbeat_pause_millis = 3000
-            first_heartbeat_estimate_millis = 1000
+            min_std_deviation = '100ms'
+            acceptable_heartbeat_pause = '3000ms'
+            first_heartbeat_estimate = '1000ms'
```
Comment (waynexia) [inline src/cmd/src/metasrv.rs:224]: same as above
```suggestion
            min_std_deviation = "100ms"
            acceptable_heartbeat_pause = "3000ms"
            first_heartbeat_estimate = "1000ms"
```

**Review by waynexia — APPROVED** [review]
LGTM

**Comment by tisonkun** [comment]
@waynexia updated.

**Pull request merged**